### DISCUSSION TITLE
Static pages and breadcrumbs

### DISCUSSION THREAD

**Discussion by ujuezuar** [OFFLINE-GmbH/oc-boxes-support#147] [Help]
As i can understand Pages created with OFFLINE.boxes do not sync with Rainlab.Pages. So i wonder what is the best way, to create nested page structure with automatically created breadcrumbs as staticBreadcrumbs do not work with pages created with OFFLINE.boxes Plugin. Any help will be appreciate :)

**Comment by tobias-kuendig** [comment]
You can access the current Boxes Page model anywhere in Twig using the `boxesPage` variable. On there you have the `getParents()` method to get all parent pages up the tree.

This is untested and needs some more code (for example to reverse the order), but something along these lines should give you the breadcrumbs:

```twig
<ul>
    {% for parent in boxesPage.getParents() %}
        <li>{{ parent.title }}</li>
    {% endfor %}
</ul>
```

It is also possible that you have to relax the Twig security policy if you want to go this route. Otherwise you'd have to move the logic to the PHP section:

https://talk.octobercms.com/t/safe-mode-relaxing-twigs-security-policy/876

Please post your solution if you get it to work!
